### ISSUE TITLE
Cannot tear tabs off to create new windows

### ISSUE THREAD

**Issue by mlissner** [eoger/tabcenter-redux#102]
STR:
1. Open a new tab
2. Load a webpage into the tab.
3. Drag the tab from the sidebar off of the browser

Expected results:
A new window should be created containing that tab.

Actual results:
The tab just goes back to where it started. 

This is a common workflow that I use and which was supported on the original tabcenter.

**Comment by eoger** [comment]
Not sure how this is handled by Firefox, maybe not possible. Needs investigation!

**Comment by jhasse** [comment]
Can you add the D-tabcenter-parity label? This issue is also very important for me.

**Comment by mlissner** [comment]
> Not sure how this is handled by Firefox, maybe not possible. Needs investigation!

FWIW, Tree Style Tabs does this.

**Comment by SmaugTheGreat** [comment]
https://github.com/piroor/treestyletab/blob/master/webextensions/sidebar/drag-and-drop.js

This is how TST does it. Looks like a lot of code. Not sure if the license allows blatantly copying it, but that would probably be the easiest solution.

**Comment by Keith94** [comment]
Previous implementation was here: 19217f9... but it was backed out due to bugs that I encounteted. So far no progress because few people have complained :P

**Comment by SmaugTheGreat** [comment]
I personally would like to see this feature implemented and may try myself on the implementation / bugfixing some time, but I think it's not really important. Just a nice to have.

**Comment by Keith94** [comment]
Yeah, I definitely implore you to give it a shot! It's been a long time since this was reported.
